include/simde/types.hpp

```diff
@@ -76,6 +76,9 @@ using braket = chemist::braket::BraKet<BraType, OpType, KetType>;
 /// Typedef of the class used to represent an atomic orbital space
 using aos = chemist::wavefunction::AOs;
 
+/// Typedef of the class used to represent a product of atomic orbital spaces
+using aos_product = chemist::dsl::Multiply<aos, aos>;
```

Comment:
```suggestion
using aos_squared = chemist::dsl::Multiply<aos, aos>;
```

I don't know if we'd ever need products of three AO spaces, but if so your typedef could be problematic.